ufs_srweather_app.settings.in

```diff
@@ -22,4 +22,4 @@ Extra libraries:	@LIBS@
 # Features
 ----------
 Parallel Support:	@HAS_PARALLEL@
-
+RRFS Components:        @ENABLE_RRFS@
```

Comment:
Just out of curiosity, do these variables (between the `@`s) correspond to the cmake variables?

Reply:
Yes.  I will say, though, that I do not have in-depth knowledge of this particular CMake feature so cannot speak with authority on details of how it works.  I needed a place to report to the user whether RRFS was enabled for the build or not.  This is where other option selections were being reported, so I added it here.  I originally had placed it in the top-level CMake, but moved it here because I thought it was more logical to be grouped with the other features.

Reply:
Ok, thanks.